README.md

```diff
@@ -135,7 +133,7 @@ public void onPermissionsDenied(int requestCode, List<String> perms) {
 public void onActivityResult(int requestCode, int resultCode, Intent data) {
     super.onActivityResult(requestCode, resultCode, data);
 
-    if (requestCode == RC_SETTINGS_SCREEN) {
+    if (requestCode == AppSettingsDialog.DEFAULT_SETTINGS_REQ_CODE) {
```

Comment:
Wouldn't it be simpler to use the default request code? The only reason I can see for setting a custom RC is if you want to use the same RC for different requests.